core/src/retransmit_stage.rs

```diff
@@ -208,7 +210,7 @@ impl RetransmitStage {
         cluster_info: &Arc<RwLock<ClusterInfo>>,
         retransmit_sockets: Arc<Vec<UdpSocket>>,
         repair_socket: Arc<UdpSocket>,
-        fetch_stage_receiver: PacketReceiver,
+        verified_receiver: CrossbeamReceiver<VerifiedPackets>,
```

Comment:
Why is it using crossbeam? 
@sakridge saw that unbounded channels on crossbeam are much slower than std::mpsc.

Reply:
@sagar-solana this what the current sigverify stage is using,
@sakridge can we switch to regular mpsc everywhere?

Reply:
Ah you're right. I think crossbeam was convenient because it's multi-producer multi-consumer. its fine if @sakridge +1s it.